lib/asm-parser.js

```diff
@@ -208,7 +242,9 @@ class AsmParser extends AsmRegex {
             asm = asm.replace(blockComments, "");
         }
 
-        const result = [];
+        const astResult = [];
```

Comment:
What's the "ast" in `astResult` here? It sounds like "abstract syntax tree" and I'm not sure it is? It's definitely better than having the `result` from before, so thank you for changing it! But I'm not sure I understand the new name.

Reply:
Oh, sorry for this, I wanted to use `asmResult` instead of `astResult`.

Now I did some renaming here:
- function parameter will be `asmResult`.
- we will collect asm values to a list called `asm`.

What do you think?